### PR TITLE
Allow coordinator to pick up existing tasks on startup

### LINKED ISSUE #1154: task: bolt store should reclaim tasks at startup
Currently, created tasks become orphaned after a restart of influxd. 

To fix this, all active tasks need to be claimed through the coordinator at startup. Should be straightforward to add after #1106 is addressed.

### PR DESCRIPTION
**Pull request by lyondhill**
Closes #1154

### CODE REVIEW

File: task/backend/coordinator/coordinator_test.go [hunk 1]
```diff
@@ -136,7 +137,7 @@ func TestCoordinator_DeleteUnclaimedTask(t *testing.T) {
 	st := backend.NewInMemStore()
 	sched := mock.NewScheduler()
 
-	coord := coordinator.New(sched, st)
+	coord := coordinator.New(zap.NewNop(), sched, st)
```
Comment (mark-rushakoff) [inline task/backend/coordinator/coordinator_test.go]: Prefer `zaptest.NewLogger(t)` in test.

File: task/backend/coordinator/coordinator.go [hunk 2]
```diff
@@ -24,20 +26,45 @@ func WithLimit(i int) Option {
...
+			}
+		}
+		tasks, err = c.Store.ListTasks(context.Background(), backend.TaskSearchParams{
+			After: tasks[len(tasks)-1].Task.ID,
+		})
+		if err != nil {
+			c.logger.Error("failed list additional tasks", zap.Error(err))
+		}
+	}
+	return
```
Comment (mark-rushakoff) [inline task/backend/coordinator/coordinator.go]: Unnecessary return.

File: task/backend/coordinator/coordinator_test.go [hunk 3]
```diff
@@ -153,3 +154,26 @@ func TestCoordinator_DeleteUnclaimedTask(t *testing.T) {
...
 	}
 }
+
+func TestCoordinator_ClaimExistingTasks(t *testing.T) {
+	st := backend.NewInMemStore()
+	sched := mock.NewScheduler()
+
+	createChan := sched.TaskCreateChan()
+
+	_, err := st.CreateTask(context.Background(), backend.CreateTaskRequest{Org: 1, User: 2, Script: script})
```
Comment (mark-rushakoff) [inline task/backend/coordinator/coordinator_test.go]: It would be better if this test had more than one "page" of tasks created.

Comment (lyondhill) [reply task/backend/coordinator/coordinator_test.go]: I thought about that. Do you think it would be worth the time to create 1000+ records?

Comment (lyondhill) [reply task/backend/coordinator/coordinator_test.go]: OO nevermind. page default is 100 :) I can do that.

**Pull request merged**